float32[3] pose
float32[2] walkingTo
float32[2] shootingTo
float32 ballAge
float32[2] ball
float32[2] ballVel
int8[5] suggestion # 5 is the maximum number of players
int8 intention
int16 averageWalkSpeed
int16 maxKickDistance
int8 currentPositionConfidence
int8 currentSideConfidence
uint16 numOfDataBytes
uint8[780] data # 780 is the standard message data size